h1st/model/fuzzy/fuzzy_model.py

```diff
@@ -30,19 +28,33 @@ def __init__(self, variables: FuzzyVariables = None, rules: FuzzyRules = None):
             )
 
     def process_rules(self, input_data: Dict) -> Dict:
-        """
+        '''
         Process rules on input_data.
 
         .. code-block:: python
             :caption: example
             input_data = {'var1': 5, 'var2': 9}
```

Comment:
in process_rules, duplicated "input_data" between the method input param and the method description block. This could make users confused when mentioning about input_data